Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -658,10 +658,17 @@ private void LogEntryDeletion(OperationContext context, ShortHash hash, ContentL
         /// <param name="contentHashListWithDeterminism">
         ///     The value, and associated determinism guarantee, to store.
         /// </param>
+        /// <param name="contentSession">
+        ///     The content session over which this operation is happening
+        /// </param>
+        /// <param name="cts">
+        ///     A token that can signal this call should return as soon as possible.
+        /// </param>
         /// <returns>
         ///     Result providing the call's completion status.
         /// </returns>
-        public abstract AddOrGetContentHashListResult AddOrGetContentHashList(OperationContext context, StrongFingerprint strongFingerprint, ContentHashListWithDeterminism contentHashListWithDeterminism);
+        public abstract Task<AddOrGetContentHashListResult> AddOrGetContentHashListAsync(OperationContext context, StrongFingerprint strongFingerprint, ContentHashListWithDeterminism contentHashListWithDeterminism, IContentSession contentSession,
+            CancellationToken cts);

```

Comment:
nit: I think the argument should be 'token' or 'ct'. I think 'cts' stands for CancellationTokenSource.

Reply:
OperationContext has a cancellationToken so this argument shouldn't be needed.

---
In reply to: [305160579](https://github.com/microsoft/BuildXL/pull/621#discussion_r305160579) [](ancestors = 305160579)